randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -266,20 +270,20 @@ Extra - map_name: rm_a3h06
   * Layers: default
   * Open Passage to Exterior Gamma Nest Access/Dock to Exterior Gamma Nest
   > Top
-      Space Jump Wall
+      Power Grip Climb
 
 > Top; Heals? False
   * Layers: default
   > Dock to Exterior Gamma Nest Access
-      Trivial
+      Morph Ball
   > Event - Gamma
       Defeat Gamma
 
 > Event - Gamma; Heals? False
   * Layers: default
   * Event Metroids Area 3 - Exterior Gamma
   > Top
-      Trivial
+      Morph Ball
```

Comment:
Why does this need morph ball?

Reply:
![image](https://github.com/randovania/randovania/assets/125271738/4c228bff-3161-4a1e-a2ee-e70f9eb67fce)
There is a one block gap you need to pass to exit.

Reply:
your image is from top -> dock
the db change i commented on is gamma -> top